Makefile

```diff
@@ -147,13 +147,24 @@ ifneq ($(BYPASS_SIGNATURES),0)
 DEFINES += HAVE_BYPASS_SIGNATURES
 endif
 
+# Enable the SET_PLUGIN test key
+SET_PLUGIN_TEST_KEY:=0
```

Comment:
Real Makefile conditional assignment is `?=`
Let's use it everywhere from now on